templates/updatedb.conf.epp

```diff
@@ -11,6 +11,12 @@ PRUNE_BIND_MOUNTS = "yes"
 <%- } else { -%>
 PRUNE_BIND_MOUNTS = "no"
 <%- } -%>
+<% if $prunefs { -%>
```

Comment:
Since the variable cannot be set to `undef` (see my above comment), the only override possible is currently to set the variables to `[]` so we should rather check if the arrays are [empty](https://www.puppet.com/docs/puppet/7/function.html#empty) (and we can revert the data type change above).

Reply:
Will make defaults `[]` again makes sense.

Reply:
Defaults are again so can be set reset to `[]` or an explicit array of choice.